src/java.base/share/classes/java/lang/System.java

```diff
@@ -2180,7 +2212,11 @@ private static void initPhase3() {
                         ctor.setAccessible(true);
                         SecurityManager sm = (SecurityManager) ctor.newInstance();
                         implSetSecurityManager(sm);
-                        needWarning = true;
+                        System.err.printf("""
+                                WARNING: A Security Manager implementation has been enabled on the command line
+                                WARNING: java.lang.SecurityManager is deprecated and will be removed in a future release
+                                WARNING: -Djava.security.manager=%s will have no effect when java.lang.SecurityManager is removed
+                                """, smProp);
```

Comment:
Raw strings may be useful here but means the lines length are inconsistent and makes it too hard to look at side by side diffs now.

Reply:
I understand what you mean when I switch to Split View.  While I can extract the lines to a method, I somehow think it's not worth doing because for each type of warning the method is only called once.